work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -1719,6 +1722,27 @@ def wait(self, timeout=WORK_QUEUE_WAITFORTASK):
         return None
 
 
+def qmap(f, a):
```

Comment:
@bwiseman77 I think we want these functions to be a method of WorkQueue, so that we can call them like:

```python
import work_queue as wq
q = wq.WorkQueue(port=9123)

fn = lambda x: x*x
results = q.map(fn, [1,2,3,4])
```

Further, in the case of `map`, we need to return the results in the same order as the arguments were given to us. I suggest keeping a dictionary of  `p_task.id -> i`, where `i` is the index of the argument in something like:

```
for (i, arg) in enumerate(args):
      p_task = PythonTask(f, arg)
      ...
```

Reply:
Ok i was thinking i needed to keep track and assumed I was getting lucky but wasnt sure the best way of keeping track.